src/components/NewSpaceCwForm.js

```diff
@@ -38,111 +39,112 @@ const NewSpaceCwForm = () => {
-        {errorMessage && <p className="error-message">{errorMessage}</p>}
-
-        <button type="submit" className="new-space-btn">Create Coworking Space</button>
-      </form>
+    <div className={styles.newCwsContainer}>
+      <div className={styles.newCwsOverlay}>
+        <h2 className={styles.newCwsTitle}>Create a New Coworking Space</h2>
+        <form onSubmit={handleSubmit} className={styles.formContainer}>
+          {/* <label htmlFor="name">Name:</label> */}
+          <input
+            type="text"
+            name="name"
+            className="text"
+            placeholder="Name:"
+            value={formData.name}
+            onChange={handleChange}
+            required
+          />
+
+          {/* <label htmlFor="model">Model:</label> */}
+          <input
+            type="text"
+            name="model"
+            className="text"
+            placeholder="Model:"
+            value={formData.model}
+            onChange={handleChange}
+          />
+
+          {/* <label htmlFor="description">Description:</label> */}
+          <textarea
+            name="description"
+            className="textarea"
+            rows="5"
+            placeholder="Description:"
+            value={formData.description}
+            onChange={handleChange}
+            required
+          />
+
+          {/* <label htmlFor="address">Address:</label> */}
+          <input
+            type="text"
+            name="address"
+            className="text"
+            placeholder="Address:"
+            value={formData.address}
+            onChange={handleChange}
+          />
+
+          {/* <label htmlFor="price">Price:</label> */}
+          <input
+            type="number"
+            name="price"
+            className="number"
+            placeholder="Price: 10.0"
+            value={formData.price}
+            onChange={handleChange}
+          />
+
+          {/* <label htmlFor="image">Image URL:</label> */}
+          <input
+            type="text"
+            name="image"
+            className="text"
+            placeholder="Image URL:"
+            value={formData.image}
+            onChange={handleChange}
+            required
+          />
+
+          {/* <label htmlFor="discount">Discount:</label> */}
+          <input
+            type="number"
+            name="discount"
+            className="number"
+            placeholder="Discount: 10.0"
+            value={formData.discount}
+            onChange={handleChange}
+          />
+
+          {/* <label htmlFor="category">Category:</label> */}
+          <input
+            type="text"
+            name="category"
+            className="text"
+            placeholder="Category:"
+            value={formData.category}
+            onChange={handleChange}
+            required
+          />
+
+          {/* You can hide user_id; set it dynamically using logged-in user; */}
```

Comment:
Hi @kazim110. Please delete these comments that suggest some actions or options.